Add connector updates to the 2201.9.0 release note (#8917)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

#### `candid` package

- Introduced support for connecting to Candid's Charity Check, Essentials and Premier REST APIs.

#### `confluent.cavroserdes` package

- Introduced support for serializing and deserializing data using Avro schemas stored in the Confluent Schmema Registry.

#### `confluent.cregistry` package

- Introduced new APIs to connect with the Confluent Schema Registry.

#### `docusign.dsadmin` package

- Enhanced the connector APIs by incorporating resource functions.
- The package name has been changed to `docusign.dsadmin` from its previous name, `docusign.admin`.

#### `docusign.dsclick` package

- Enhanced the connector APIs by incorporating resource functions.
- The package name has been changed to `docusign.dsclick` from its previous name, `docusign.click`.

#### `docusign.dsesign` package

- Introduced new APIs to connect with DocuSign eSignature REST APIs.

#### `github` package

- Introduced support for connecting to GitHub REST API version 2022-11-28, replacing the GitHub GraphQL API based connector.

#### `googleapis.gcalendar` package

- Enhanced the connector APIs by incorporating resource functions.

> - The package name has been changed to `googleapis.gcalendar` from its previous name, `googleapis.calendar`.

Reviewer: HEre, we have to mention that the previous package is also available.

### Revamped connector updates

The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative.

> All the connectors listed below have been released under new major versions, due to significant changes in the APIs and functionalities.
> These updates also include improvements to the documentation and examples.

Reviewer 2: ```suggestion
All listed connectors have been released under new major versions, featuring significant API and functionality changes, along with improved documentation and examples.
```

### Revamped connector updates

The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative.
All the connectors listed below have been released under new major versions, due to significant changes in the APIs and functionalities.

#### `asana` package

- Enhanced connector APIs by incorporating resource function syntax, along with improved documentation and examples.

#### `candid` package

- Introduced support for connecting to Candid's Charity Check, Essentials and Premier REST APIs.

#### `confluent.cavroserdes` package

- Introduced support for serializing and deserializing data using Avro schemas stored in the Confluent Schmema Registry.

#### `confluent.cregistry` package

- Introduced support for integrating with the Confluent Schema Registry.

#### `docusign.dsadmin` package

- Enhanced the connector APIs to align with DocuSign Admin APIs by incorporating resource functions.
- With the new changes the package name is changed to `docusign.dsadmin` which was previously maintained under the name `docusign.admin`.

> - Add the documentation and examples.

Reviewer 3: Is this relevant to the release note? There are a few occurrences of this.

Reviewer 2: I also don't think we need to specifically mention about the doc improvements, which is common for all the revamped connectors(in that case we'll end up repeating the same point under every connector).

Reviewer: We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

Reviewer 2: > We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

yeah that makes more sense. +1 for the suggestion.

#### `docusign.dsclick` package

- Enhanced the connector APIs to align with DocuSign Click APIs by incorporating resource functions.
- With the new changes the package name is changed to `docusign.dsclick` which was previously maintained under the name `docusign.click`.
- Add the documentation and examples.

#### `docusign.dsesign` package

- Introduced new APIs to align with DocuSign eSignature APIs by incorporating resource functions.
- Add the documentation and examples.

#### `github` package

- Introduced support for connecting to GitHub REST API version 2022-11-28, replacing the GitHub GraphQL API based connector.

#### `googleapis.gcalendar` package

- Enhanced the connector APIs by incorporating resource functions.

> - With the new changes the package name is changed to `googleapis.gcalendar` which was previously maintained under the name `googleapis.calendar`.

Reviewer 3: ```suggestion
The package name has been changed to `googleapis.gcalendar` from its previous name, `googleapis.calendar`.
```
